src/components/SingleReview.jsx

```diff
@@ -17,9 +21,10 @@ const SingleReview = () => {
     }
 
     useEffect(() => {
-        fetchReviewById(review_id)
-        .then((reviewFromApi) => {
+        Promise.all([fetchReviewById(review_id), fetchCommentsByReviewId(review_id)])
```

Comment:
We should fetch the comments in the Comments component

Reply:
commit message - [moved fetching comments to Comments component]